vcf/model.py

```diff
@@ -117,6 +117,22 @@ def is_het(self):
             return None
         return self.gt_type == 1
 
+    @property
+    def is_filt(self):
+        """ Return True for filtered calls """
+        try: # no FT annotation present for this variant
+            FT=self.data.FT
+        except AttributeError:
+            return False
+        if FT == None or FT == []: # FT is not set or set to PASS
```

Comment:
Compare to `None` with `is` (not `==`) and check for the empty list with `len(FT) == 0`.